src/pages/guides/FAQ/index.md

```diff
@@ -1,38 +1,47 @@
-# Frame V4 API FAQ
+# Frame.io API FAQ
 
-The Frame V4 API allows developers to integrate Frame.io’s surface for creatives and their teams to manage their work.
+The Frame.io API allows developers to integrate Frame.io’s surface for creatives and their teams to manage their work.
 
-* Where is the **API reference**?
-    * Go [here](https://developer-stage.adobe.com/frameio/api/current/) to see the Frame V4 API reference pages.
-* How can I get **help**?
-    * Reach out to the Frame.io support team for assistance by chatting with us on our [Support Site](https://help.frame.io/en/). Please note that enterprise customers may be directed to their Customer Success Manager.
-* What are the **key concepts** to the Frame V4 API?
-    * V4 represents an expansion from primarily serving post-production video professionals to a discipline-agnostic platform for creatives and creative managers. Frame.io acts as a central canvas for multi-disciplinary teams and solves for key needs for everything from more focus time for creative work to increased visibility into ongoing projects.
-    * **Workflow management**: Centralize, organize, and build all your end-to-end creative workflows at scale leveraging Metadata, Custom Fields, and Collections.
-    * **File transfer and management**: This set of capabilities gives users all the flexibility and power to build the best file management system for their creative process making it easier to quickly get the information they need, and manage all their work-in-progress assets at scale.
-    * **Review and approval**: Frame.io helps creative teams reduce the total number of revision cycles with advanced review, commenting, and annotation capabilities available across a broad set of media types.
-    * **Sharing and presentation**: Customize and share creative work from one central place.
-    * **Security and governance**: View and manage all users, workspaces, and projects in one place. Take bulk actions and export information to control how you utilize usage data.
-    * **Open API Standards**: A REST API and an OpenAPI specification connect Frame.io to the systems you already use to unblock custom workflows.
+### Where is the **API reference**?
 
-* Is the Frame V4 API **backward compatible**?  
+Go [here](https://developer-stage.adobe.com/frameio/api/current/) to see the Frame V4 API reference pages.
 
-    No. The Frame product released at Adobe MAX operates in fundamentally different ways and emphasizes different workflows than the legacy version. Due to these differences, the new API is not backward compatible.
+### How can I get **help**?
```

Comment:
@chandlma switched these to `###` headings so they'll display with the sidebar shortcuts (see screenshot in PR)